### PR TITLE
[ODS-5033] Create Template builds for TPDM Core

### PR DESCRIPTION
**Pull request by raghuramakrishna**

### CODE REVIEW

**Review by tbmeador-dlp — CHANGES_REQUESTED** [review]

File: DatabaseTemplate/Modules/create-tpdm-minimal-template.psm1 [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        ("$(Get-PluginFolderFromSettings $config.appSettings)\EdFi.Suite3.Ods.Extensions.TPDM*\Artifacts\Schemas\")
+    )
+
+    $config.databaseBackupName = "EdFi.Ods.Minimal.Template.TPDM"
+    $config.packageNuspecName = "EdFi.Ods.Minimal.Template.TPDM"
+
+    return $config
+}
+
+Set-Alias -Scope Global initmin Initialize-MinimalTemplate
```
[the remark this answers is not in the document]
Comment (raghuramakrishna) [reply DatabaseTemplate/Modules/create-tpdm-minimal-template.psm1]: removed

File: DatabaseTemplate/Modules/create-tpdm-minimal-template.psm1 [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        PS> Initialize-PopulatedTempalate -samplePath "C:\edfi\Ed-Fi-Standard\v3.2\"
+    #>
+    param(
+        [Parameter(
+            Mandatory = $false,
+            HelpMessage = "An absolute path to the folder to load samples from, for example: C:\MySampleXmlData\.`r`nAlso supports specific version folders of the Data Standard repository, for example: C:\Ed-Fi-Standard\v3.0\ or C:\Ed-Fi-Standard\v2.0\"
+        )]
+        [ValidateNotNullOrEmpty()]
+        [ValidateScript( { Resolve-Path $_ } )]
+        [string] $samplePath = "$PSScriptRoot/../../../Ed-Fi-TPDM-Extension/",
```
[the remark this answers is not in the document]
Comment (raghuramakrishna) [reply DatabaseTemplate/Modules/create-tpdm-minimal-template.psm1]: updated

File: DatabaseTemplate/Modules/create-tpdm-minimal-template.psm1 [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+    $config.Remove('bulkLoadTempDirectorySample')
+    $config.bulkLoadBootstrapInterchanges = @("InterchangeDescriptors")
+    $config.bulkLoadMaxRequests = 1
+    $config.schemaDirectories = @(
+        (Get-RepositoryResolvedPath "Application\EdFi.Ods.Standard\Artifacts\Schemas\")
+        ("$(Get-PluginFolderFromSettings $config.appSettings)\EdFi.Suite3.Ods.Extensions.TPDM*\Artifacts\Schemas\")
+    )
+
+    $config.databaseBackupName = "EdFi.Ods.Minimal.Template.TPDM"
+    $config.packageNuspecName = "EdFi.Ods.Minimal.Template.TPDM"
```
[the remark this answers is not in the document]
Comment (raghuramakrishna) [reply DatabaseTemplate/Modules/create-tpdm-minimal-template.psm1]: updated

File: DatabaseTemplate/Modules/create-tpdm-minimal-template.psm1 [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+        Ignores any extension sources when running the sql scripts against the database.
+
+    .PARAMETER noValidation
+        Disables xml validation.
+
+    .parameter Engine
+    The database engine provider, either 'SQLServer' or 'PostgreSQL'
+
+    .EXAMPLE
+        PS> Initialize-PopulatedTempalate -samplePath "C:\edfi\Ed-Fi-Standard\v3.2\"
```
[the remark this answers is not in the document]
Comment (raghuramakrishna) [reply DatabaseTemplate/Modules/create-tpdm-minimal-template.psm1]: updated

**Review by tbmeador-dlp — APPROVED** [review]

**Pull request merged**